proxies/live/apiproxy/targets/target.xml

```diff
@@ -18,7 +18,7 @@
       </Step>
       <Step>
           <Condition>request.header.NHSD-Connection-Metadata = null</Condition>
```

Comment:
Shouldn't this have a different condition for the nrl app id?

Reply:
no so the way it works is that it does that connection metadata flow and if it returns before setting the connection metadata header this will be triggered. It's how we used to do it for when we have no pointers. Unfortunately apigee doesn't pass in the app custom attributes through flows so cant do "app.nrl-app-id = null" as a condition, either we set it as a header or something then use it but that isnt great and we don't really want it as a header.

Reply:
Ah gotcha. My question here is where it triggers this error in cases where it shouldn't? Like something else not set, but then it returns back with this error? I guess the answer of this is not as long as we don't do any other early return, right?